sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -102,6 +102,10 @@ public UserDto selectActiveUserByLogin(DbSession session, String login) {
     return executeLargeInputs(logins, session.getMapper(UserMapper.class)::selectByLogins);
   }
 
+  /**
+   * @deprecated since 6.0 please use {@link #selectByLogins(DbSession, Collection)} instead
+   */
+  @Deprecated
   public List<UserDto> selectByLogins(Collection<String> logins) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)